submissions/yaripey/dom-task/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="css/main.css">
+  <script src="js/main.js"></script>
+</head>
+
+
+<body>
+
+  <header>
+    <a href="#" class="toggle"></a>
+    <img class="logo" src="img/logo.png" alt="logo">
+  </header>
+
+  <div class="middle">
```

Comment:
Please use more informative class names. `middle` is too generic name, to my mind.

Reply:
Changed this particular one, do you see any other classes too generic?